pom.xml

```diff
@@ -56,6 +56,7 @@
 	</dependencies>
 
 	<build>
+		<defaultGoal>spring-boot:run</defaultGoal>
```

Comment:
The BaseBuild Element Set
defaultGoal: the default goal or phase to execute if none is given. If a goal is given, it should be defined as it is in the command line (such as jar:jar). This case, it is spring-boot:run.